src/jit/regset.cpp

```diff
@@ -2339,8 +2339,6 @@ regNumber RegSet::rsUnspillOneReg(GenTreePtr tree, regNumber oldReg, KeepReg wil
 //
 TempDsc* RegSet::rsUnspillInPlace(GenTreePtr tree, regNumber oldReg, unsigned regIdx /* =0 */)
 {
-    assert(!isRegPairType(tree->gtType));
```

Comment:
I am not sure this is the right fix.  May be I am don't have the full context here.

On x86 RyuJIT a multi-reg call node is not considered a regpair type.  In fact with multi-reg call node support added to x86 ryujit we have disabled regPair types.  We need to understand  why gentree is considered a repair type.